Constants.h

```diff
@@ -122,8 +122,8 @@ const unsigned long KI_MAX = 10;
 const unsigned long DEADZONE = 0;
 const unsigned long MIN_POS = -100;
 const unsigned long MAX_POS = 700;
-const unsigned long VEL_MAX = 1800;     // Maximum velocity (clicks/s) to command
-const unsigned long ACC_MAX = 200000;   // Maximum acceleration (clicks/s^2) to command
+const unsigned long VEL_MAX = 1800;     // Maximum velocity (ticks/s) to command  (this equals approx. 20 motor RPM)
```

Comment:
Comments should not assume a particular value of the constant because the comment becomes confusing if we change the value in the future. Also, we've been trying to avoid lines longer than 100 columns for readability.
```suggestion
const unsigned long VEL_MAX = 1800;     // Maximum velocity (ticks/s) to command
                                        // (one motor RPM equals approx. 90 clicks)
```

Reply:
Done -  I added // 1 Rev = 5264 ticks (7pulse*4clicks*188 Gear Ratio)